### PR TITLE
FEATURE: Adds per day report of reactions & likes

### PR DESCRIPTION
**Pull request by Ahmedgagan**
@jjaffeux

### CODE REVIEW

**Comment by jjaffeux** [comment]
I wonder if we could find a way to override likes_report when reactions is enabled

**Comment by Ahmedgagan** [comment]
I'll take a look if it's possible 😅

**Comment by Ahmedgagan** [comment]
@jjaffeux Does the latest commit look okay? I tried to override the likes_report method

**Comment by jjaffeux** [comment]
Looks nice 👍  now try to separate likes and custom reactions please

File: plugin.rb [hunk 1]
```diff
@@ -143,4 +143,85 @@ class Engine < ::Rails::Engine
...
+            }
+          ]
+
+          titles = DiscourseReactions::Reaction.where("reaction_users_count IS NOT NULL AND reaction_value!=?", DiscourseReactions::Reaction.main_reaction_id).pluck(:reaction_value).uniq
+
+          titles.each { |title|
+            report.labels << {
+              type: :number,
+              property: "#{title}_count",
+              title: title
```
Comment (jjaffeux) [inline plugin.rb]: Cant you just do this here:

```
PrettyText.unescape_emoji(CGI::escapeHTML(title))
```

Comment (Ahmedgagan) [reply plugin.rb]: I tried it, But it doesn't change anything. It returns the same string even if the emoji is not available

**Comment by jjaffeux** [comment]
You must be doing something wrong:

```
[1] pry(main)> PrettyText.unescape_emoji(CGI::escapeHTML(":+1:"))
=> "<img width=\"20\" height=\"20\" src='/images/emoji/twitter/+1.png?v=9' title='+1' alt='+1' class='emoji'>"
```

Like do you have : as prefix and suffix ?

**Comment by Ahmedgagan** [comment]
> You must be doing something wrong:
> 
> 
> 
> ```
> 
> [1] pry(main)> PrettyText.unescape_emoji(CGI::escapeHTML(":+1:"))
> 
> => ""
> 
> ```
> 
> 
> 
> Like do you have : as prefix and suffix ?

Yes i was forgetting :

**Comment by Ahmedgagan** [comment]
@jjaffeux This worked also added change to the core PR to support this change

**Comment by Ahmedgagan** [comment]
@jjaffeux made it compatible with new core changes

**Comment by jjaffeux** [comment]
I already did it locally :) There are multiple things not working here that I'm trying to fix.

**Pull request closed without merging**